ci/cli-test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -ex
+
+bundle install
+
+${@-bundle exec rake jenkins}
```

Comment:
`$@` gets set to all the command line arguments. Using `-` in a parameter substitution uses the value of the parameter if it's set, or the string after the `-` it's not.

So, this will either run the arguments as the command, or run `bundle exec rake jenkins` if no arguments are passed.

This script gets set as the entrypoint of the test container. So, if you run `docker-compose run test`, it runs all the tests. If you run `docker-compose run test /bin/bash`, it puts you in a shell.